try.html

```diff
@@ -534,6 +534,10 @@ <h3 id="version"> Version </h3>
   <td><img src='/github/tag/expressjs/express.svg' alt=''/></td>
   <td><code>https://img.shields.io/github/tag/expressjs/express.svg</code></td>
   </tr>
+  <tr><th data-doc='githubDoc'> GitHub development: </th>
```

Comment:
How about **GitHub package.json version**?

Could you add one for `/github/manifest-json` too?

Reply:
Good idea, will be sure to do this.